scripts/AbinsModules/LoadCRYSTAL.py

```diff
@@ -335,41 +340,17 @@ def _parse_item(self, item=None, container=None, part=None):
         else:
             raise ValueError("Real or imaginary part of complex number was expected.")
 
-    def _move_to(self, file_obj=None, msg=None):
-        """
-        Finds the first line with msg. Moves file current position to the found line line which includes .
-        :param file_obj: file object from which we read
-        :param msg: keyword to find
-        """
-        while not self._file_end(file_obj=file_obj):
-            pos = file_obj.tell()
-            line = file_obj.readline()
-            if line.strip() and msg in line:
-                file_obj.seek(pos)
-                return
-
-    def _find(self, file_obj=None, msg=None):
-        """
-        Finds the first line with msg. Moves file current position to the next line.
-        :param file_obj: file object from which we read
-        :param msg: keyword to find
-        :return: line with the msg keyword
-        """
-        while not self._file_end(file_obj=file_obj):
-            line = file_obj.readline()
-            if line.strip() and msg in line:
-                return line
-
     def _check_kpoints_end(self, file_obj=None):
         """
-
+        Checks if end of k-points block.
         :param file_obj: file object from which we read
         :return: True if end of block otherwise False
         """
-        allowed_keywords = [" X ", " Y ", " Z ", "-", "REAL", "IMAGINARY", "MODES", "DISPERSION"]
+        allowed_keywords = [b" X ", b" Y ", b" Z ", b"-", b"REAL", b"IMAGINARY", b"MODES", b"DISPERSION"]
+
         # remove empty lines:
         pos = None
-        while not self._file_end(file_obj=file_obj):
+        while not self._parser.file_end(file_obj=file_obj):
             pos = file_obj.tell()
             line = file_obj.readline()
             if line.strip():
```

Comment:
Line 364-367 could be rewritten with `return any(key in line for key in allowed_keywords)`